core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -573,6 +578,94 @@ public Journaled getDelegate() {
     return mJournaledGroup;
   }
 
+  @Override
+  public JournalContext createJournalContext() throws UnavailableException {
+    JournalContext context = super.createJournalContext();
+    if (!Configuration.getBoolean(
+        PropertyKey.MASTER_FILE_SYSTEM_MERGE_INODE_JOURNALS
```

Comment:
Probably all related properties should be static finals? I think it is better that they can't be changed.

Reply:
k will change